README.md

```diff
@@ -133,7 +133,7 @@ Client client = Clients.builder()
     .setAuthorizationMode(AuthorizationMode.PRIVATE_KEY)
     .setClientId("{clientId}")
     .setScopes(new HashSet<>(Arrays.asList("okta.users.read", "okta.apps.read")))
-    .setPrivateKey("/path/to/yourPrivateKey.pem")
+    .setPrivateKey("/path/to/yourPrivateKey.pem") // (or) .setPrivateKey("full PEM payload");
```

Comment:
IMHO, if we are releasing a 3.0 (different PR)  we should have `.setPrivateKey(File/Path)` and `.setPrivateKey(String pemContent)` This would also allow us to add `.setPrivateKey(java.security.PrivateKey)` easily in the future.  Some of the other Okta SDKs also support a JWKS object as well.

Reply:
noted.

Reply:
That would break backwards compatibility quite subtly because there is already a `setPrivateKey` that takes a String-argument.

Reply:
Agreed, there is another PR with other breaking changes (which is why I suggested this here)

Reply:
We have a `v3.0.0` in the making. Will release this breaking change as part of v3.

cc: @sergiishamrai-okta